ui/ui-components/components/navigation.tsx

```diff
@@ -158,7 +159,7 @@ export default function Navigation(props) {
               </a>
             </Link>
             <br />
-            <Link href="/settings/[tab]" as="/settings/core">
+            <Link href={settingsLink} as={settingsLinkAs}>
```

Comment:
I recall you mentioned you were going to remove the settings page. Did you do that? If so, might want to disable this link. Based on our workflow, I'd say it's safe to assume that it will be complete by the time the user is viewing it, since they have to register before they can view this page.

That is, unless they call their company "My Grouparoo Cluster"

Reply:
Good call.  I had it linking to '`'/'` instead of `'/settings/core'` on config mode... but I think disabling it makes more sense.